Services/Storage/DocumentDb/DocumentDbRecord.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Exceptions;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Storage.DocumentDb
+{
+    public class DocumentDbRecord : Resource
+    {
+        public const long NEVER = -1;
+
+        public string Data { get; set; }
+        public long ExpirationUtcMsecs { get; set; }
+        public long LastModifiedUtcMsecs { get; set; }
+        public string LockOwnerId { get; set; }
+        public string LockOwnerType { get; set; }
+        public long LockExpirationUtcMsecs { get; set; }
+        private static long Now => DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
```

Comment:
nit: move this after line 11, with empty line separating. We have some unofficial way of sorting class elements, e.g. public constants first, then private fields, then public properties, ctor, public methods, private methods